index.js

```diff
@@ -291,10 +305,14 @@ Use lotsOfArt to do the following:
 
 For example lotsOfArt(artists); will return ["Amedeo Modigliani", "Rene Magritte", ... "Albrecht Dürer"]*/
 
-function lotsOfArt(/*Your Code Here*/){
-  /*Your Code Here*/
+function lotsOfArt(array){
+  for (let i = 0; i > 100; i++){
+    if (artists[i].paintings > 100){
```

Comment:
You are hard coding the artists array on line 310 and 311.  You'll want to make use of the function parameter called array in your function.  The basic steps for this problem are to start with making a new empty array called artistsOverOneHundred = [];  ... Then you can do your for loop, and populate your new array.  Return the new array from the function.